Source/BugsnagConfiguration.h

```diff
@@ -142,16 +138,46 @@ BugsnagBreadcrumbs *breadcrumbs;
 @property BOOL autoNotify;
 
 /**
- * Determines whether app sessions should be tracked automatically. By default this value is false.
+ * Determines whether app sessions should be tracked automatically. By default this value is true.
  */
 @property BOOL shouldAutoCaptureSessions;
 
 /**
- * Set the endpoint to which tracked sessions reports are sent. This defaults to https://sessions.bugsnag.com,
- * but should be overridden if you are using Bugsnag On-premise, to point to your own Bugsnag endpoint.
+ * Retrieves the endpoint used to notify Bugsnag of errors
+ *
+ * NOTE: it is strongly recommended that you set this value via setEndpointsForNotify:sessions: instead.
```

Comment:
is it worth using something like [this](https://stackoverflow.com/a/3908766/549238) to make this as deprecated?

Reply:
That's one of the outstanding questions - what are your thoughts on this? If we add `setEndpointsForNotify:sessions:`, then developers are able to set the value, but won't be able to read it without `notifyURL` and `sessionsURL`.

The way I see it there are a couple of options:

- Leave `notifyURL` and `sessionsURL` as is and document they shouldn't be set
- Deprecate then remove the properties
- Make the properties readonly (breaking change)

I've currently leant towards the first option.

Reply:
I see the dilemma. Is it possible to split the property up and only deprecate the write? I only jumped on this as I saw in the android notifier that we were deprecating bits like this as part of similar changes. If it doesn't make sense for `bugsnag-cocoa` then that is cool

Reply:
Might be worth trying: